lib/features/quiz/screen/quiz_screen.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_learn_app/features/quiz/model/quiz.dart';
+import 'package:flutter_learn_app/features/theory/screens/theory_screen.dart';
+
+class QuizScreen extends StatefulWidget {
+  const QuizScreen({super.key});
+  @override
+  State<QuizScreen> createState() => _QuizScreenState();
+}
+
+class _QuizScreenState extends State<QuizScreen> {
+  List<Question> questionList = getQuestions();
+  int currentQuestionIndex = 0;
+  int score = 0;
+  Answer? selectedAnswer;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color.fromARGB(255, 55, 50, 80),
+      body: Container(
```

Comment:
Padding
Don't use Container only for margin, padding purposes, it's much heavier than Padding